proto/interchain_security/ccv/v1/ccv.proto

```diff
@@ -9,6 +9,14 @@ import "cosmos/staking/v1beta1/staking.proto";
 import "gogoproto/gogo.proto";
 import "tendermint/abci/types.proto";
 
+// This packet is to send for preCCV chain, and once it's received consumer chain sends VSCMaturedPacketData
+// and consumer chain becomes preCCV chain as false, and provider chain record preCCV field as false on provider chain
+message ReplaceWholeValidatorSet {
```

Comment:
I don't think this msg is needed. See https://github.com/cosmos/ibc/blob/main/spec/app/ics-028-cross-chain-validation/overview_and_basic_concepts.md#channel-initialization-existing-chains